src/screens/DataTable.js

```diff
@@ -708,6 +708,13 @@ const DataTablePage = () => (
           <GenericExtend />
         </Property>
 
+        <Property name="dataTable.body.row.extend">
+          <Description>
+            Any additional style for the rows in the DataTable body.
```

Comment:
```suggestion
            Any additional CSS for the rows in the DataTable body.
```
Our theme has a mix of component props and actual CSS. What do you think of pointing out this is CSS and not Box props?

Reply:
yes seems helpful